WalletWasabi.Gui/Tabs/WalletManager/RecoverWallets/RecoverWalletView.xaml

```diff
@@ -5,55 +5,69 @@
-  <StackPanel Margin="10" Spacing="10">
-    <controls:GroupBox Title="{Binding Title}" TextBlock.FontSize="30" Padding="30">
-      <Grid Classes="content">
-        <StackPanel Spacing="8">
-          <StackPanel>
-            <TextBlock Text="Heads Up!" FontWeight="Bold" />
-            <TextBlock Text="At recovery, the wallet is unable to check if your password is correct or not. If you provide a wrong password, a wallet will be recovered with the provided Recovery Words and password combination." TextWrapping="Wrap" />
-          </StackPanel>
-          <controls:ExtendedTextBox Text="{Binding WalletName}" Watermark="Wallet Name" UseFloatingWatermark="True" />
-          <controls:TogglePasswordBox Text="{Binding Password}">
-            <i:Interaction.Behaviors>
-              <behaviors:CommandOnEnterBehavior Command="{Binding RecoverCommand}" />
-            </i:Interaction.Behaviors>
-          </controls:TogglePasswordBox>
-          <controls:ExtendedTextBox Text="{Binding MnemonicWords}" Watermark="Recovery Words" UseFloatingWatermark="True" CaretIndex="{Binding CaretIndex}">
-            <i:Interaction.Behaviors>
-              <suggestions:SuggestionBehavior SuggestionItems="{Binding Suggestions}" />
-            </i:Interaction.Behaviors>
-          </controls:ExtendedTextBox>
-          <ItemsControl Items="{Binding Suggestions}">
-            <ItemsControl.ItemsPanel>
-              <ItemsPanelTemplate>
-                <StackPanel Orientation="Horizontal" Spacing="2" />
-              </ItemsPanelTemplate>
-            </ItemsControl.ItemsPanel>
-            <ItemsControl.ItemTemplate>
-              <DataTemplate>
-                <Button Focusable="False" Content="{Binding Word}" Command="{Binding OnSelected}" IsPressed="{Binding IsHighLighted}" />
-              </DataTemplate>
-            </ItemsControl.ItemTemplate>
-          </ItemsControl>
-          <StackPanel Spacing="16">
-            <CheckBox IsChecked="{Binding ShowAdvancedOptions, Mode=TwoWay}">
-              <TextBlock Text="Show Advanced Options" />
-            </CheckBox>
-            <StackPanel IsVisible="{Binding ShowAdvancedOptions}">
-              <controls:ExtendedTextBox Text="{Binding AccountKeyPath}" Watermark="Account Key Path" UseFloatingWatermark="True" />
-              <TextBlock Text="Wasabi can only monitor native SegWit (bech32) addresses." Classes="warningMessage" />
+  <ScrollViewer x:Name="Root" HorizontalScrollBarVisibility="Hidden">
+    <Border Width="{Binding #Root.Bounds.Width}" Padding="{StaticResource PageRootPadding}">
+      <StackPanel Spacing="10">
+        <StackPanel.Styles>
+          <Style Selector="Expander /template/ ToggleButton#PART_toggle /template/ Border">
+            <Setter Property="Background" Value="Transparent" />
+          </Style>
+        </StackPanel.Styles>
+        <controls:GroupBox Title="{Binding Title}" TextBlock.FontSize="30" Padding="15 0">
```

Comment:
IMO this is better:
```suggestion
        <controls:GroupBox Title="{Binding Title}" TextBlock.FontSize="30" Padding="30">
```
and it will be consistent with other tabs like `Generate Wallet` tab.

Reply:
we now have extra width coming from the containing panel... we can not add height here either.. so `30 0` could be acceptable. Id vote to keep it as it is.